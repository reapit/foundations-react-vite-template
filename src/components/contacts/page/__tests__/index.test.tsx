import ContactsPage from '..'
import { render } from '../../../../scripts/tests'

describe('ContactsPage', () => {
  it('should match a snapshot', () => {
    expect(render(<ContactsPage />)).toMatchSnapshot()
  })
})
