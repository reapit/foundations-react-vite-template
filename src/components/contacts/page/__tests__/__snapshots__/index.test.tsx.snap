// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactsPage should match a snapshot 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="el-snack-holder"
      />
      <div
        class="el-flex-auto el-flex-container"
      >
        <aside
          class="el-secondary-nav-container"
        >
          <h1
            class="el-title"
          >
            Contacts
          </h1>
          <div
            class="el-mb9 el-secondary-nav"
          >
            <div
              class="el-secondary-nav-item"
            >
              List
            </div>
            <div
              class="el-secondary-nav-item"
            >
              New
            </div>
          </div>
          <span
            class="el-mb5 el-icon-size-large el-icon"
          >
            <svg
              fill="none"
              height="1em"
              viewBox="0 0 80 80"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <mask
                height="80"
                id="leadGenerationInfographic_svg__a"
                maskUnits="userSpaceOnUse"
                style="mask-type: alpha;"
                width="81"
                x="0"
                y="0"
              >
                <circle
                  fill="#F2F2F2"
                  r="40"
                  transform="matrix(1 0 .00002 1 40 40)"
                />
              </mask>
              <g
                mask="url(#leadGenerationInfographic_svg__a)"
              >
                <path
                  d="M0 0h80v80H0z"
                  fill="#d8dee4"
                />
                <path
                  d="M18 72.797c2.109-4.588 4.873-8.94 8.247-12.196 2.746-2.65 8.533-3.95 9.75-3.474 2.19-3.02 4.989-6.127 4.989-6.127s4.971.965 8.026 5.984a.226.226 0 0 1 .021-.03c.685-.77 2.338-.296 6.048 1.31 4.12 1.784 5.716 5.122 7.138 8.094.339.708.668 1.395 1.018 2.036A77.81 77.81 0 0 0 65 71.444c-5.774 6.114-13.68 10.069-22.543 10.514-9.364.47-18.025-3.064-24.457-9.16Z"
                  fill="#798da1"
                />
                <path
                  d="M28.95 80c-.66-.263-1.31-.549-1.95-.856 1.339-3.538 3.131-7.544 3.131-7.544S29.055 75.79 28.95 80ZM57 77.318c-.558.353-1.127.69-1.706 1.008-.535-4.012-1.862-7.751-1.862-7.751s1.95 3.468 3.568 6.743ZM42.384 61.797c3.446-.182 6.133-2.743 6.002-5.722-.131-2.979-3.031-5.246-6.477-5.065-3.447.181-6.134 2.743-6.003 5.722.132 2.978 3.032 5.246 6.478 5.065Z"
                  fill="#9faebc"
                />
              </g>
              <path
                d="M8.931 34.591a3.83 3.83 0 0 0 3.23-1.785c.634-.992.626-1.87.62-2.575l-.001-.2.014-.035c-3.337-1.407-5.683-4.755-5.683-8.663C2.743 2.51 27.834 5.376 27.834 18.713a2.313 2.313 0 0 1-2.323 2.324v.296c0 .213-.007.424-.021.633a9.333 9.333 0 0 1-2.318 5.611 9.224 9.224 0 0 1-3.344 2.419l.014.035v.194c-.006.722-.013 1.592.618 2.58a3.83 3.83 0 0 0 3.23 1.786C21.5 37.5 12 37.5 8.931 34.591Z"
                fill="#FFE5C7"
              />
              <path
                d="M3.2 9.368a.06.06 0 0 0 .01.07 5.863 5.863 0 0 1 1.623 4.06v5.092c0 1.373 1.09 2.491 2.443 2.491V15.85c0-.515.487-.885.969-.733 1.515.475 3.282.72 5.14.308 3.268-.724 4.194-2.03 6.86-2.684 2.979-.728 5.68 1.6 5.662 4.577-.006.978-.022 2.202-.024 3.763a2.34 2.34 0 0 0 2.35-2.352v-3.984c.03-1.387-.032-6.279-3.496-9.685C20.709 1.1 14.326 1.425 9.703 3.543a.194.194 0 0 0 .006.353l1.206.506c.18.076.142.347-.051.368l-.586.064c-3.15.344-5.771 2.261-7.078 4.534Z"
                fill="#506478"
              />
              <path
                d="M2 41.518v3.827C7.5 51 25 51 30.622 45.345v-3.827c0-4.015-3.104-7.268-6.931-7.27-1.354 0-2.545-.73-3.231-1.838-2.393 2.145-5.904 2.146-8.298 0-.686 1.107-1.878 1.838-3.231 1.839C5.103 34.25 2 37.503 2 41.518Z"
                fill="#9faebc"
              />
              <path
                d="M56.931 34.591a3.83 3.83 0 0 0 3.23-1.785c.634-.992.626-1.87.62-2.575l-.001-.2.014-.035c-3.337-1.407-5.683-4.755-5.683-8.663-4.368-18.823 20.723-15.957 20.723-2.62a2.313 2.313 0 0 1-2.323 2.324v.296c0 .213-.007.424-.021.633a9.333 9.333 0 0 1-2.318 5.611 9.224 9.224 0 0 1-3.344 2.419l.014.035v.194c-.007.722-.013 1.592.618 2.58a3.83 3.83 0 0 0 3.23 1.786C69.5 37.5 60 37.5 56.932 34.591Z"
                fill="#FFE5C7"
              />
              <path
                d="M51.2 9.368a.06.06 0 0 0 .01.07 5.863 5.863 0 0 1 1.623 4.06v5.092c0 1.373 1.09 2.491 2.443 2.491V15.85c0-.515.487-.885.969-.733 1.515.475 3.282.72 5.14.308 3.268-.724 4.194-2.03 6.86-2.684 2.979-.728 5.68 1.6 5.662 4.577-.006.978-.022 2.202-.024 3.763a2.34 2.34 0 0 0 2.35-2.352v-3.984c.03-1.387-.032-6.279-3.496-9.685-4.028-3.96-10.411-3.635-15.034-1.517a.194.194 0 0 0 .006.353l1.206.506c.18.076.142.347-.051.368l-.586.064c-3.15.344-5.771 2.261-7.078 4.534Z"
                fill="#506478"
              />
              <path
                d="M50 41.518v3.827c5.5 5.655 23 5.655 28.622 0v-3.827c0-4.015-3.104-7.268-6.931-7.27-1.354 0-2.545-.73-3.231-1.838-2.393 2.145-5.904 2.146-8.298 0-.686 1.107-1.878 1.838-3.231 1.839C53.103 34.25 50 37.503 50 41.518Z"
                fill="#9faebc"
              />
              <path
                d="M24.204 37.893c-.824.882-1.308 2.121-1.243 3.47.122 2.537 2.142 4.501 4.51 4.387.092-.004.184-.012.274-.022 1.789 5.684 5.772 12.837 14.46 12.42 9.664-.466 13.268-7.707 14.402-14.097 2.092-.393 3.625-2.4 3.513-4.735a4.79 4.79 0 0 0-1.053-2.794c-.567 1.334-2.083.796-2.628-.49-.46-1.085-.534-2.223-.607-3.363-.096-1.498-.193-3.001-1.168-4.403-2.91-4.184-5.273-3.603-9.033-2.678-1.283.316-2.73.672-4.415.892-5.184.676-10.369-2.015-10.369-2.015S27.134 29.85 27.695 38.3c.13 1.959-2.335 2.025-3.491-.407Z"
                fill="#FFE5C7"
              />
              <path
                d="M34.138 33.088c-.423.07-.93.154-.906-.253.16-2.815 6.383-1.724 5.496-.078-.185.343-.647.108-1.134-.14-.336-.172-.684-.349-.96-.346-1.156.01-1.494.265-1.795.492-.184.14-.354.268-.69.324l-.01.001ZM49.653 31.327c.974-.108 2.38.234 2.805 1.029.213.398.033.54-.491.688-.45.126-.75-.037-1.074-.214-.097-.053-.196-.107-.303-.155a2.546 2.546 0 0 0-1.265-.208 1.897 1.897 0 0 0-.604.17c-.141.057-.29.12-.491.165-.298.068-.681.114-.699-.141-.059-.835 1.307-1.244 2.122-1.334ZM51.082 34.962a1.475 1.475 0 0 0-.224-.104c-2.61-.958-3.495 3.176-1.59 3.847 2.75.969 3.447-2.81 1.814-3.743ZM36.457 34.858c.079.029.153.064.224.104 1.633.933.935 4.712-1.814 3.743-1.905-.67-1.02-4.805 1.59-3.847Z"
                fill="#506478"
              />
              <path
                d="M50.76 35.42c-.29-.274-.84-.03-.762.343.04.19.078.343.254.473.269.198.806-.034.784-.35-.014-.206-.126-.323-.277-.466Z"
                fill="#fff"
              />
              <path
                d="M36.065 48.077s6.424 3.832 11.32 1.01c-2.655 3.83-9.828 3.244-11.32-1.01ZM28.135 17.605c3.597-5.204 11.183-6.418 18.044-5.132 3.884.728 16.638 5.131 12.991 12.604.213 2.238.333 4.495.335 6.744 0 1.43.028 2.966-.326 4.363-.442 1.75-2.153 1.232-2.74-.151-1.066-2.512-.057-5.297-1.775-7.767-3.903-5.612-6.824-2.65-13.448-1.786-5.184.676-10.368-2.015-10.368-2.015S27.133 29.85 27.695 38.3c.177 2.666-4.453 1.826-4.048-4.044.405-5.87-.997-16.124 4.488-16.65Z"
                fill="#506478"
              />
              <path
                d="M36.358 35.42c-.29-.274-.84-.03-.761.343.04.19.078.343.254.472.268.199.805-.033.784-.35-.015-.205-.126-.322-.277-.465Z"
                fill="#fff"
              />
              <path
                d="M33.507 59.2a19.593 19.593 0 0 1-2.773-.664c-6.438-2.063-10.148-7.283-11.457-14.063-.124-.642-1.044-.626-1.146.02C16.886 52.42 12.38 58.09 4.464 59.717c-.63.13-.614 1.049.02 1.156 7.894 1.333 12.676 6.829 14.192 14.645.095.492.611.485.582.474a.584.584 0 0 0 .564-.496c.63-4.055 2.506-9.582 7.853-12.883 1.708-1.053 3.678-1.813 5.854-2.257a.586.586 0 0 0 .466-.587.586.586 0 0 0-.488-.57Z"
                fill="#7e9bfa"
              />
            </svg>
          </span>
          <p
            class="el-has-grey-text el-small-text"
          >
            This app provides the full search, view, edit and delete capabilities for your contacts.
          </p>
          <p
            class="el-has-grey-text el-small-text"
          >
            For more information, please click the link below.
          </p>
          <button
            class="el-mb5 el-intent-neutral el-button"
          >
            <div
              class="el-button-loader"
            />
            Docs
          </button>
        </aside>
        <section
          class="el-hfull el-pb11 el-page-container"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="el-snack-holder"
    />
    <div
      class="el-flex-auto el-flex-container"
    >
      <aside
        class="el-secondary-nav-container"
      >
        <h1
          class="el-title"
        >
          Contacts
        </h1>
        <div
          class="el-mb9 el-secondary-nav"
        >
          <div
            class="el-secondary-nav-item"
          >
            List
          </div>
          <div
            class="el-secondary-nav-item"
          >
            New
          </div>
        </div>
        <span
          class="el-mb5 el-icon-size-large el-icon"
        >
          <svg
            fill="none"
            height="1em"
            viewBox="0 0 80 80"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <mask
              height="80"
              id="leadGenerationInfographic_svg__a"
              maskUnits="userSpaceOnUse"
              style="mask-type: alpha;"
              width="81"
              x="0"
              y="0"
            >
              <circle
                fill="#F2F2F2"
                r="40"
                transform="matrix(1 0 .00002 1 40 40)"
              />
            </mask>
            <g
              mask="url(#leadGenerationInfographic_svg__a)"
            >
              <path
                d="M0 0h80v80H0z"
                fill="#d8dee4"
              />
              <path
                d="M18 72.797c2.109-4.588 4.873-8.94 8.247-12.196 2.746-2.65 8.533-3.95 9.75-3.474 2.19-3.02 4.989-6.127 4.989-6.127s4.971.965 8.026 5.984a.226.226 0 0 1 .021-.03c.685-.77 2.338-.296 6.048 1.31 4.12 1.784 5.716 5.122 7.138 8.094.339.708.668 1.395 1.018 2.036A77.81 77.81 0 0 0 65 71.444c-5.774 6.114-13.68 10.069-22.543 10.514-9.364.47-18.025-3.064-24.457-9.16Z"
                fill="#798da1"
              />
              <path
                d="M28.95 80c-.66-.263-1.31-.549-1.95-.856 1.339-3.538 3.131-7.544 3.131-7.544S29.055 75.79 28.95 80ZM57 77.318c-.558.353-1.127.69-1.706 1.008-.535-4.012-1.862-7.751-1.862-7.751s1.95 3.468 3.568 6.743ZM42.384 61.797c3.446-.182 6.133-2.743 6.002-5.722-.131-2.979-3.031-5.246-6.477-5.065-3.447.181-6.134 2.743-6.003 5.722.132 2.978 3.032 5.246 6.478 5.065Z"
                fill="#9faebc"
              />
            </g>
            <path
              d="M8.931 34.591a3.83 3.83 0 0 0 3.23-1.785c.634-.992.626-1.87.62-2.575l-.001-.2.014-.035c-3.337-1.407-5.683-4.755-5.683-8.663C2.743 2.51 27.834 5.376 27.834 18.713a2.313 2.313 0 0 1-2.323 2.324v.296c0 .213-.007.424-.021.633a9.333 9.333 0 0 1-2.318 5.611 9.224 9.224 0 0 1-3.344 2.419l.014.035v.194c-.006.722-.013 1.592.618 2.58a3.83 3.83 0 0 0 3.23 1.786C21.5 37.5 12 37.5 8.931 34.591Z"
              fill="#FFE5C7"
            />
            <path
              d="M3.2 9.368a.06.06 0 0 0 .01.07 5.863 5.863 0 0 1 1.623 4.06v5.092c0 1.373 1.09 2.491 2.443 2.491V15.85c0-.515.487-.885.969-.733 1.515.475 3.282.72 5.14.308 3.268-.724 4.194-2.03 6.86-2.684 2.979-.728 5.68 1.6 5.662 4.577-.006.978-.022 2.202-.024 3.763a2.34 2.34 0 0 0 2.35-2.352v-3.984c.03-1.387-.032-6.279-3.496-9.685C20.709 1.1 14.326 1.425 9.703 3.543a.194.194 0 0 0 .006.353l1.206.506c.18.076.142.347-.051.368l-.586.064c-3.15.344-5.771 2.261-7.078 4.534Z"
              fill="#506478"
            />
            <path
              d="M2 41.518v3.827C7.5 51 25 51 30.622 45.345v-3.827c0-4.015-3.104-7.268-6.931-7.27-1.354 0-2.545-.73-3.231-1.838-2.393 2.145-5.904 2.146-8.298 0-.686 1.107-1.878 1.838-3.231 1.839C5.103 34.25 2 37.503 2 41.518Z"
              fill="#9faebc"
            />
            <path
              d="M56.931 34.591a3.83 3.83 0 0 0 3.23-1.785c.634-.992.626-1.87.62-2.575l-.001-.2.014-.035c-3.337-1.407-5.683-4.755-5.683-8.663-4.368-18.823 20.723-15.957 20.723-2.62a2.313 2.313 0 0 1-2.323 2.324v.296c0 .213-.007.424-.021.633a9.333 9.333 0 0 1-2.318 5.611 9.224 9.224 0 0 1-3.344 2.419l.014.035v.194c-.007.722-.013 1.592.618 2.58a3.83 3.83 0 0 0 3.23 1.786C69.5 37.5 60 37.5 56.932 34.591Z"
              fill="#FFE5C7"
            />
            <path
              d="M51.2 9.368a.06.06 0 0 0 .01.07 5.863 5.863 0 0 1 1.623 4.06v5.092c0 1.373 1.09 2.491 2.443 2.491V15.85c0-.515.487-.885.969-.733 1.515.475 3.282.72 5.14.308 3.268-.724 4.194-2.03 6.86-2.684 2.979-.728 5.68 1.6 5.662 4.577-.006.978-.022 2.202-.024 3.763a2.34 2.34 0 0 0 2.35-2.352v-3.984c.03-1.387-.032-6.279-3.496-9.685-4.028-3.96-10.411-3.635-15.034-1.517a.194.194 0 0 0 .006.353l1.206.506c.18.076.142.347-.051.368l-.586.064c-3.15.344-5.771 2.261-7.078 4.534Z"
              fill="#506478"
            />
            <path
              d="M50 41.518v3.827c5.5 5.655 23 5.655 28.622 0v-3.827c0-4.015-3.104-7.268-6.931-7.27-1.354 0-2.545-.73-3.231-1.838-2.393 2.145-5.904 2.146-8.298 0-.686 1.107-1.878 1.838-3.231 1.839C53.103 34.25 50 37.503 50 41.518Z"
              fill="#9faebc"
            />
            <path
              d="M24.204 37.893c-.824.882-1.308 2.121-1.243 3.47.122 2.537 2.142 4.501 4.51 4.387.092-.004.184-.012.274-.022 1.789 5.684 5.772 12.837 14.46 12.42 9.664-.466 13.268-7.707 14.402-14.097 2.092-.393 3.625-2.4 3.513-4.735a4.79 4.79 0 0 0-1.053-2.794c-.567 1.334-2.083.796-2.628-.49-.46-1.085-.534-2.223-.607-3.363-.096-1.498-.193-3.001-1.168-4.403-2.91-4.184-5.273-3.603-9.033-2.678-1.283.316-2.73.672-4.415.892-5.184.676-10.369-2.015-10.369-2.015S27.134 29.85 27.695 38.3c.13 1.959-2.335 2.025-3.491-.407Z"
              fill="#FFE5C7"
            />
            <path
              d="M34.138 33.088c-.423.07-.93.154-.906-.253.16-2.815 6.383-1.724 5.496-.078-.185.343-.647.108-1.134-.14-.336-.172-.684-.349-.96-.346-1.156.01-1.494.265-1.795.492-.184.14-.354.268-.69.324l-.01.001ZM49.653 31.327c.974-.108 2.38.234 2.805 1.029.213.398.033.54-.491.688-.45.126-.75-.037-1.074-.214-.097-.053-.196-.107-.303-.155a2.546 2.546 0 0 0-1.265-.208 1.897 1.897 0 0 0-.604.17c-.141.057-.29.12-.491.165-.298.068-.681.114-.699-.141-.059-.835 1.307-1.244 2.122-1.334ZM51.082 34.962a1.475 1.475 0 0 0-.224-.104c-2.61-.958-3.495 3.176-1.59 3.847 2.75.969 3.447-2.81 1.814-3.743ZM36.457 34.858c.079.029.153.064.224.104 1.633.933.935 4.712-1.814 3.743-1.905-.67-1.02-4.805 1.59-3.847Z"
              fill="#506478"
            />
            <path
              d="M50.76 35.42c-.29-.274-.84-.03-.762.343.04.19.078.343.254.473.269.198.806-.034.784-.35-.014-.206-.126-.323-.277-.466Z"
              fill="#fff"
            />
            <path
              d="M36.065 48.077s6.424 3.832 11.32 1.01c-2.655 3.83-9.828 3.244-11.32-1.01ZM28.135 17.605c3.597-5.204 11.183-6.418 18.044-5.132 3.884.728 16.638 5.131 12.991 12.604.213 2.238.333 4.495.335 6.744 0 1.43.028 2.966-.326 4.363-.442 1.75-2.153 1.232-2.74-.151-1.066-2.512-.057-5.297-1.775-7.767-3.903-5.612-6.824-2.65-13.448-1.786-5.184.676-10.368-2.015-10.368-2.015S27.133 29.85 27.695 38.3c.177 2.666-4.453 1.826-4.048-4.044.405-5.87-.997-16.124 4.488-16.65Z"
              fill="#506478"
            />
            <path
              d="M36.358 35.42c-.29-.274-.84-.03-.761.343.04.19.078.343.254.472.268.199.805-.033.784-.35-.015-.205-.126-.322-.277-.465Z"
              fill="#fff"
            />
            <path
              d="M33.507 59.2a19.593 19.593 0 0 1-2.773-.664c-6.438-2.063-10.148-7.283-11.457-14.063-.124-.642-1.044-.626-1.146.02C16.886 52.42 12.38 58.09 4.464 59.717c-.63.13-.614 1.049.02 1.156 7.894 1.333 12.676 6.829 14.192 14.645.095.492.611.485.582.474a.584.584 0 0 0 .564-.496c.63-4.055 2.506-9.582 7.853-12.883 1.708-1.053 3.678-1.813 5.854-2.257a.586.586 0 0 0 .466-.587.586.586 0 0 0-.488-.57Z"
              fill="#7e9bfa"
            />
          </svg>
        </span>
        <p
          class="el-has-grey-text el-small-text"
        >
          This app provides the full search, view, edit and delete capabilities for your contacts.
        </p>
        <p
          class="el-has-grey-text el-small-text"
        >
          For more information, please click the link below.
        </p>
        <button
          class="el-mb5 el-intent-neutral el-button"
        >
          <div
            class="el-button-loader"
          />
          Docs
        </button>
      </aside>
      <section
        class="el-hfull el-pb11 el-page-container"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
